openvino_xai/methods/black_box/aise.py

```diff
@@ -19,8 +22,9 @@
 from openvino_xai.methods.black_box.base import BlackBoxXAIMethod, Preset
 
 
-class AISE(BlackBoxXAIMethod):
-    """AISE explains classification models in black-box mode using
+class AISEBase(BlackBoxXAIMethod, ABC):
```

Comment:
What do you think about creating `AISE` folder and splitting this big file into 
`AISE`
- `base.py`
- `classification.py`
- `detection`?

Reply:
This is definitely an option, I am neutral. What do you think @goodsong81?

Reply:
I'm OK with the directory strategy.